frontend/src/components/Page4AllPaths.module.css

```diff
@@ -0,0 +1,174 @@
+.page4Form {
+  background-color: white;
+  box-shadow: 0 5px 10px rgba(0, 0, 0, 0.25);
+}
+
+.formRow {
+  display: flex;
+  justify-content: space-between;
+  margin-left: 100px;
+  margin-right: 130px;
+  padding-bottom: 30px;
+}
+
+@media (max-width: 1250px) {
+  .formRow {
+    display: flex;
+    flex-direction: column;
+  }
+  .formRow .input {
+    width: 330px;
+  }
+}
+
+.sectionTitle {
+  display: flex;
+  font-family: "Rubik";
+  font-size: 18px;
+  font-weight: 700;
+  margin-left: 100px;
+  margin-top: 0.25em;
+  margin-bottom: 30px;
+  padding-top: 2em;
+}
+
+.label {
+  display: flex;
+  flex-direction: column;
+  align-items: left;
+  font-family: "Rubik";
+  font-size: 12px;
+  font-weight: 200;
+  color: #6c6c6c;
+}
+
+.input {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 40px;
+  width: 460px;
+  border-radius: 4px;
+  border: 1px solid #d8d8d8;
+  font-family: "Rubik";
+  font-size: 16px;
+  font-weight: 200;
+}
+
+.input::placeholder {
+  text-indent: 1em;
+  color: var(--color-ccidc-placeholder);
+}
+
+.input option {
+  color: var(--color-ccidc-placeholder);
+}
+
+.line {
+  border: 0.5px solid #d8d8d8;
+  margin-top: 30px;
+}
+
+.bottomLine {
+  border: 0.5px solid rgba(216, 216, 216, 0.25);
+  margin-top: 30px;
+}
+
+.note {
+  margin-left: 100px;
+}
+
+.lightBlue {
+  color: var(--color-ccidc-blue);
+}
+
+.warningText {
+  color: #ad281f;
+  margin-left: 100px;
+  margin-top: 15px;
+  margin-bottom: 40px;
+}
+
+.upload {
+  font-family: "Rubik";
+  font-size: 15px;
+  background-color: white;
+  color: black;
+  border: 1px solid black;
+  border-radius: 4px;
+  height: 40px;
+  width: 285px;
+  text-align: left;
+  align-items: center;
+  display: flex;
+  margin-left: 100px;
+  background-image: url("../assets/uploadIcon.svg");
+  background-repeat: no-repeat, repeat;
+  background-position:
+    right 2em top 50%,
+    0 0;
+  margin-bottom: 2em;
+}
+
+input[type="checkbox"].select {
+  margin-left: 0;
+  visibility: hidden;
```

Comment:
This would mess with the requirement seeting.

The red color  of the checked box can be achieved through attribute `accent-color: red;` just give the appropriate color.
Size of the checkbox can be changed using width and height.  You can keep the hover.